CHANGES/8971.bugfix

```diff
@@ -0,0 +1 @@
+Renaming bindings to be compatible with pulpcore >= 3.14
```

Comment:
Do we need to version bump our pulpcore requirement due to this?

Reply:
I think we do

Reply:
yes, the name change happened on pulpcore 3.14 - https://github.com/pulp/pulpcore/pull/1438